### ISSUE TITLE
The miner should be slashed but not

### ISSUE THREAD

**Issue by steven004** [filecoin-project/go-filecoin#3474]
**Describe the bug**:
I stopped my go-filecoin daemon yesterday, and restart it after about 2000 blocks. I would expect my power will come to 0 due to the slash mechanism, but it is not. I still have power, and I could mine continuously, while my proving period keeps as the origin one.

**Expected behavior**:
My power is slashed to 0. 

**Diagnostic information:**
```
$ ./go-filecoin miner power t2ekqlkmh73knwnsjcsqsozyor7tbrnpu4x5ejcfi
268435456 / 805306368
$ ./go-filecoin mining status
Mining Status
Active:     true
Address:    t2ekqlkmh73knwnsjcsqsozyor7tbrnpu4x5ejcfi
Owner:      t1tkj7rpejskqvavivfhs3oevw76su4j55qdgkavq
Collateral: 0.012
Power:      268435456 / 805306368

Proving Period
Start:         5742
End:           5892
Proving Set:   [1 2 3 4 5 6 7]
```
*As it shows, I stopped my program before block height 5742, and restart it around block height 7471.* 

- Filecoin Version: staging-devnet-0.5.3
- Filecoin Inspect Output: 
```
$ ./go-filecoin inspect all
Version:	a4c3f704ec726122e3460f276a252a57ae14e509

Runtime
OS:           	darwin
Arch:         	amd64
Version:      	go1.12.4
Compiler:     	gc
NumProc:      	12
GoMaxProcs:   	12
NumGoRoutines:	366
NumCGoCalls:  	19152

Disk
Free:  	146489626624
Total: 	141477040128
FSType:	24

Memory
Swap:   	0
Virtual:	0
```

**Comment by mishmosh** [comment]
Thanks for the report. @anorth will investigate.

**Comment by steven004** [comment]
Agreed to move on, and this can be fixed later if it happens again. 

One more question: 
> Thanks for the report. Not being slashed means that no other miner, that actually produced a block **in the available period**, attempted to do so.

@anorth , do you mean a miner just can be slashed in a fixed period in the current implementation? while after the period, it will not be slashed any more? I actually run other nodes without change the slash mechanism, even so, this particular node did not get slashed either.

**Comment by anorth** [comment]
> in the available period
I mean until your miner submitted a new PoSt and thus became good again. There's no facility for retroactive slashing at the moment.

> I actually run other nodes
Unless we know that those nodes mined a block during the right period, this doesn't tell us much.
